admin_manual/installation/deployment_recommendations.rst

```diff
@@ -377,6 +377,19 @@ You may also encounter database errors, such as this one:
 The issue, `identified by Michael Roth`_, is caused when MariaDB Galera cluster sends write requests to all servers in the cluster; `here is a detailed explanation`_.
 The solution is to send all write requests to a single server, instead of all of them.
 
+---------
+
+Set the wsrep_sync_wait parameter to 1 on all nodes of your Galera Cluster. It is disabled per default.
```

Comment:
- `wsrep_sync_wait` should be delimited as a code block, to make it that much clearer.
- s/per default/by default/g